label_studio/frontend/src/pages/PeoplePage/SelectedUser.js

```diff
@@ -0,0 +1,67 @@
+import { format } from "date-fns";
+import { NavLink } from "react-router-dom";
+import { LsCross } from "../../assets/icons";
+import { Button, Userpic } from "../../components";
+import { Block, Elem } from "../../utils/bem";
+import "./SelectedUser.styl";
+
+const UserProjectsLinks = ({projects}) => {
+  return (
+    <Elem name="links-list">
+      {projects.map((project) => (
+        <Elem tag={NavLink} name="project-link" key={`project-${project.id}`} to={`/projects/${project.id}`}>
+          {project.title}
+        </Elem>
+      ))}
+    </Elem>
+  );
+};
+
+export const SelectedUser = ({ user, onClose }) => {
+  const fullName = [user.first_name, user.last_name].filter(n => !!n).join(" ").trim();
+
+  return (
```

Comment:
what about this compound design element?

![image](https://user-images.githubusercontent.com/6087484/112219589-42192980-8c36-11eb-8089-dcf8d54e2d9f.png)